pkg/chartverifier/checks/charttesting.go

```diff
@@ -107,6 +111,11 @@ func ChartTesting(opts *CheckOptions) (Result, error) {
 		return NewResult(false, err.Error()), nil
 	}
 
+	configRelease := opts.ViperConfig.GetString(ReleaseConfigString)
+	if len(configRelease) >0 {
```

Comment:
```suggestion
	if len(configRelease) > 0 {
```

`go fmt` check is failing.

Reply:
oops.